gulp/modules/ErrorHandling.js

```diff
@@ -1,16 +1,233 @@
 var path = require('path');
 var gulputil = require('gulp-util');
+var map = require('map-stream');
+var Config = require('./Config');
 /**
  * Custom Error Handling Class for Various Tasks
  */
 var ErrorHandling = function() {
     /**
+     * Stores error messages
+     */
+    var errors = [];
+    /**
+     * Stores Warning Messages
+     */
+    var warnings = [];
+    var that = this;
```

Comment:
We might consider refactoring in the future using the revealing module pattern so we don't have to do 'that' for 'this' as often. Or moving forward using the revealing module pattern.